knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -681,3 +683,14 @@ func executeStages(instance *serverlessoperatorv1alpha1.KnativeKafka, manifest *
 	}
 	return nil
 }
+
+func removeCreationTimestamp(manifest *mf.Manifest, _ *serverlessoperatorv1alpha1.KnativeKafka) error {
+	// Avoid multiple unnecessary resource updates and reconciler looping
+	// due to creationTimestamp set to `null` on some resources.
```

Comment:
When `null` is set? Could you use a predicate to filter these changes on meta instead?

Reply:
I'm using the same approach the other functions are using, see 
https://github.com/openshift-knative/serverless-operator/blob/26a310ee58a38e70d3f7bd1d1d057891d06e287f/knative-operator/pkg/controller/knativekafka/images.go#L86-L88
(there are many other instances of the same logic)